pkgs/development/libraries/wolfssl/default.nix

```diff
@@ -10,7 +10,7 @@
 # requiring to build a special variant for that software. Example: 'haproxy'
 , variant ? "all"
 , extraConfigureFlags ? []
-, enableLto ? !(stdenv.isDarwin || stdenv.hostPlatform.isStatic || stdenv.cc.isClang)
+, enableLto ? !(stdenv.hostPlatform.isStatic || stdenv.cc.isClang)
```

Comment:
still not enabled

Reply:
I don’t know, which is why I left that part of the condition in place. If the clang part is resolved, then Darwin should just work.